n_threads: 4  # number of threads to use.

r_min: 0.5 # minimum point distance.
r_max: 200 # maximum point distance.
n_bins: 400 # number of radial (range from sensor) bins per segment.
n_segments: 360 # number of radial segments.

max_dist_to_line: 0.2 # maximum vertical distance of point to line to be considered ground.

max_slope: 0.1  # maximum slope of a ground line.
max_fit_error: 0.1  # maximum error of a point during line fit.
long_threshold: 5.0 # distance between points after which they are considered far from each other.
max_long_height: 0.2  # maximum height change to previous point in long line.
max_start_height: 0.4 # maximum difference to estimated ground height to start a new line.
sensor_height: 1.85 # sensor height above ground.
line_search_angle: 0.15 # how far to search in angular direction to find a line [rad].

latch: true # latch output topics or not
visualize: false  # visualize segmentation result - USE ONLY FOR DEBUGGING
